scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2109,39 @@ def visit_call(self, node):
                     break
 
 
+class NoBlankLineBelowFunctionDefinition(checkers.BaseChecker):
+    """Checks if there are no blank lines available below function
+    definition.
+    """
+
+    _implements_ = interfaces.IAstroidChecker
+    name = 'newline-below-function-definition'
+    priority = -1
+    msgs = {
+        'W0033': (
+            'A blank line found below the function definition.',
+            'newline-below-function-definition',
+            'All functions must have no spaces below function definition.'
+        )
+    }
+
+    def check_blank_lines_below_function_definition(self, node):
```

Comment:
Did you explore options other than a raw checker? You take a look in our [wiki](https://github.com/oppia/oppia/wiki/Lint-Checks#python) and might find a way of implementing the same via some `IAstroidChecker ` node like [FunctionDef](http://pylint.pycqa.org/projects/astroid/en/latest/api/astroid.nodes.html#astroid.nodes.FunctionDef). Those checks would run faster than raw checkers.

Reply:
Also when you are writing a raw checker, you have name the method as `process_module` to raw parse and process an individual file ( see [eg](https://github.com/oppia/oppia/blob/develop/scripts/linters/pylint_extensions.py#L1421) here). The current method won't even be called by pylint. I just wanted to give you this additional info, and I still want you to work on suggestions in prev comment first before coming to a raw checker. Do let me know if you have any questions.

Reply:
Done!

Reply:
I have used Function node for the task.

Reply:
I have used scoped node function for this task.